packages/affirm-integration/src/affirm.ts

```diff
@@ -94,6 +94,6 @@ export interface AffirmAddress {
 }
 
 export interface AffirmScripts {
-    PROD: string;
-    SANDBOX: string;
+    prod: string;
+    sandbox: string;
```

Comment:
As I mentioned before in `affirm-script-loader.ts`

```suggestion
    export enum AFFIRM_SCRIPTS {
            PROD = '//cdn1.affirm.com/js/v2/affirm.js',
            SANDBOX =  '//cdn1-sandbox.affirm.com/js/v2/affirm.js',
    }
```